.build/docker/_docker_run.sh

```diff
@@ -118,11 +122,11 @@ docker_command="export ANT_OPTS=\"-Dbuild.dir=\${DIST_DIR} ${CASSANDRA_DOCKER_AN
 # run without the default seccomp profile
 # re-use the host's maven repository
 container_id=$(docker run --name ${container_name} -d --security-opt seccomp=unconfined --rm \
-    -v "${cassandra_dir}":/home/build/cassandra -v ~/.m2/repository/:/home/build/.m2/repository/ -v "${build_dir}":/dist \
+    -v "${cassandra_dir}":/home/build/cassandra -v ${m2_dir}:/home/build/.m2/repository/ -v "${build_dir}":/dist \
```

Comment:
Inside the `docker/run-tests.sh` we are mapping these params in the following way:
```
docker_mounts="-v 
${cassandra_dir}:/home/cassandra/cassandra -v
"${build_dir}":/home/cassandra/cassandra/build -v 
${m2_dir}:/home/cassandra/.m2/repository"
```

and here the mapping is different, should it be the same?

Reply:
there are intended differences here yes.
(maybe we could look at the /dist approach for the build_dir, but that's out-of-scope for this ticket)